index.html

```diff
@@ -4,10 +4,161 @@
+            </li>
+            <li>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Numquam,
+              velit?
+            </li>
+            <li>
+              Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ipsum,
+              amet?
+            </li>
+          </ul>
+        </div>
+      </article>
+
+      <section class="social-cards">
+        <h2 class="center-headline">Social Media</h2>
+        <div class="flex-container">
+          <div class="card">
+            <a href="https://github.com/">
+              <figure>
+                <img
+                  height="200"
+                  width="200"
+                  src="https://play-lh.googleusercontent.com/PCpXdqvUWfCW1mXhH1Y_98yBpgsWxuTSTofy3NGMo9yBTATDyzVkqU580bfSln50bFU"
+                  alt="Shows the Github logo"
+                />
+                <figcaption>Github</figcaption>
+              </figure>
+            </a>
+          </div>
+          <div class="card">
+            <a href="https://codepen.io/">
+              <figure>
+                <img
+                  height="200"
+                  width="200"
+                  src="https://media.licdn.com/dms/image/C560BAQGKtd1slGeUMA/company-logo_200_200/0/1564001544623?e=2147483647&v=beta&t=hxHAT6zqF3yYJLxhRHWSQgWH9y40xhk0kWerZ3LGxI8"
+                  alt="Shows the Github logo"
+                />
+                <figcaption>Codepen</figcaption>
+              </figure>
+            </a>
+          </div>
+          <div class="card">
+            <a href="https://de.linkedin.com/">
+              <figure>
+                <img
+                  height="200"
+                  width="200"
+                  src="https://www.svgrepo.com/show/922/linkedin.svg"
+                  alt="Shows the Github logo"
+                />
+                <figcaption>Linkedin</figcaption>
+              </figure>
+            </a>
+          </div>
+        </div>
+      </section>
+
+      <article class="about">
+        <h2 class="center-headline">About me</h2>
+        <div class="flex-container">
+          <p>
+            Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quaerat
+            harum reiciendis blanditiis iusto inventore fugiat provident tempora
+            doloremque perferendis ipsam, asperiores laudantium eveniet dolor
+            magnam dolorum hic dicta tempore excepturi reprehenderit similique
+            minima. Eum maxime minima quaerat neque ex! Neque, sed maxime
+            placeat a quaerat eum odit aliquid aut nesciunt!
+          </p>
+          <img
+            height="300"
+            width="auto"
+            src="https://www.cvplaza.com/wp-content/uploads/2013/10/example-of-a-good-cv.png"
+            alt="Shows my cv"
+          />
+        </div>
+      </article>
+
+      <section class="contact">
+        <h2>Kontakt</h2>
+
+        <form action="">
+          <label for="name">Name:</label><br />
```

Comment:
```suggestion
          <label for="name">Name:</label>
```
Lieber CSS verwenden statt br:)
aber sehr geil, dass du label eingebaut hast!